src/models/searchservice/request/UniversalSearchRequest.ts

```diff
@@ -16,6 +16,8 @@ export interface UniversalSearchRequest {
   queryTrigger?: QueryTrigger;
   /** Disables spellcheck if true. */
   skipSpellCheck?: boolean;
+  /** The session ID*/
```

Comment:
nit: missing a space after 'ID'

Reply:
also, for the documentation in this repo, we always put a period at the end because it works better in the generated docs